layouts/shortcodes/rule-id.html

```diff
@@ -0,0 +1,8 @@
+{{- $ruleId := .Inner -}}
+{{- $shortenedRuleId := print "..." (substr $ruleId -8) }}
```

Comment:
```suggestion
{{- $shortenedRuleId := print "..." (substr (trim $ruleId " ") -8) }}
```
Just in case the inner content includes spaces.